### ISSUE TITLE
Using external repos in integration tests and examples

### ISSUE THREAD

**Issue by php-coder** [openshift/source-to-image#546]
S2I uses one personal repo in integration tests:

```
test/integration/integration_test.go:   TestSource          = "git://github.com/pmorie/simple-html"
```

Also we're using personal repositories in a different examples:

```
docs/cli.md:$ s2i build git://github.com/mfojtik/sinatra-app-example openshift/ruby-20-centos7 ruby-app
docs/cli.md:$ s2i build --ref=my-branch git://github.com/mfojtik/sinatra-app-example openshift/ruby-20-centos7 ruby-app
docs/cli.md:$ s2i build --scripts-url=file://s2iscripts --destination=/opt git://github.com/mfojtik/sinatra-app-example openshift/ruby-20-centos7 ruby-app
README.md:$ s2i build git://github.com/pmorie/simple-ruby openshift/ruby-20-centos7 test-ruby-app
README.md:$ s2i build git://github.com/bparees/openshift-jee-sample openshift/wildfly-100-centos7 test-jee-app
README.md:$ s2i build git://github.com/pmorie/simple-ruby openshift/ruby-20-centos7 test-ruby-app
```

Should we create a separate repo for all of them to reduce dependency on personal repositories? We can't be sure that all these repos are up-to-date with newest versions. Also from the user's perspective I trust more to non-personal repositories.

See also: https://github.com/openshift/origin/issues/9899

**Comment by bparees** [comment]
the pmorie repo should bre placed with this:
https://github.com/openshift/ruby-hello-world

we should do the same for all the other ruby examples in the doc. (will need to test, obviously).

for the JEE example, we can probably use one of the middleware quickstarts:
https://github.com/jboss-developer/jboss-eap-quickstarts/tree/7.1.x-develop/kitchensink

**Comment by coreydaley** [comment]
Linked to #688

**Comment by coreydaley** [comment]
Fixed via #688
